### ISSUE TITLE
Missing set key,value for godot.core.Dictionary

### ISSUE THREAD

**Issue by 2shady4u** [utopia-rise/godot-kotlin#31]
Currently it is impossible to add key,value pairs to any godot.core.Dictionary as it is not implemented in the Dictionary.kt

**Minimal reproduction project:**

```
var something = Dictionary()
var key = "wakanda"
something[key] = "forever"
```
Won't work...

**Comment by piiertho** [comment]
Indeed, no set or operator is implemented in this class. We will fix this.

**Comment by piiertho** [comment]
solved by #32
